packages/cli/generators/import-lb3-model/model-names.js

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+module.exports = {
+  canImportModelName,
+};
+
+// List of built-in LB models to exclude from the prompt
+const EXCLUDED_MODEL_NAMES = [
```

Comment:
I am a bit lost here. In the docs, it says
> you can modify your LB3 model to inherit from `Model`,
>  `PersistedModel` or `KeyValueModel`, then import the model and finally update
>  the generated model

So the `EXCLUDED_MODEL_NAMES` here means, if a model extends from the list, can it be imported or not?

Reply:
> if a model extends from the list, can it be imported or not?

TL;DR: These two aspects are orthogonal. If a model extends from `Model`, `PersistedModel` or `KeyValueModel` (all of them are in the list), then it can be imported. If it inherits from a different model class, then it cannot be imported yet.

We have two list of model names:

1. List of model names to hide from the CLI prompt when asking which app models to import. This list is `EXCLUDED_MODEL_NAMES` plus any model name starting with `AnonymousModel_`.

    The purpose of this filter is to hide models that cannot be imported (e.g. `Model` and `PersistedModel` are built-in in both LB3 and LB4) or which does not make sense to import (e.g. `Email`).

    The filter does not take into account model inheritance.

    I suppose I could improve the filter to take into account model inheritance. The difficult part is how to explain the user why some models are included in the list and some are not. I would prefer to invest our time into implementing import of models inheriting from other app models instead.

2. List of base models that we know how to handle at LB4 side, see https://github.com/strongloop/loopback-next/blob/42ec9ed2b972fec9ae25b974b65c268a6b2ed34b/packages/cli/generators/import-lb3-model/migrate-model.js#L120-L124

    I created this list by picking built-in LB3 models that have their direct counter-part in LB4.

I'll be opening a new GH issue to implement support for importing models that are inheriting from a model that's not built into LB4. There are two major use cases - import from an LB3 built-in model like `User` and import from an application-specific model. There are few difficult aspects to figure out, that's why I am leaving this feature out of scope of the initial version.
- How to know where to import the base model from (what path to use in the `import` statement)?
- How to ensure that the base LB3 model has been either already imported to the LB4 app or is part of the import being in progress?